libc/cmake/modules/LLVMLibCCompileOptionRules.cmake

```diff
@@ -241,17 +241,18 @@ function(_get_common_test_compile_options output_var c_test flags)
       list(APPEND compile_options "-ffixed-point")
     endif()
 
-    # list(APPEND compile_options "-Wall")
-    # list(APPEND compile_options "-Wextra")
+    list(APPEND compile_options "-Wall")
+    list(APPEND compile_options "-Wextra")
     # -DLIBC_WNO_ERROR=ON if you can't build cleanly with -Werror.
     if(NOT LIBC_WNO_ERROR)
-      # list(APPEND compile_options "-Werror")
+      list(APPEND compile_options "-Werror")
+      list(APPEND compile_options "-Wno-global-constructors")
```

Comment:
Do you actually need to disable `-Wno-global-constructors`? It's commented out below, so I don't think it gets set?

Reply:
Good catch, I think I accidentally set this when initially attempting this issue. Removing